### PR TITLE
added port-fowarding link, how to op/run commands, and "server cant keep up error"

### PR DESCRIPTION
**Pull request by cyric2020**

### CODE REVIEW

**Review by vaio2005 — APPROVED** [review]
All should be fine.

**Comment by Fiahblade** [comment]
@cyric2020 Looks great, can you fix the conflicts so that I can merge it? Thanks!

**Comment by Fiahblade** [comment]
Not sure if you noticed but there is a forge installation section: https://mcserversoft.github.io/documentation/create-forge-server/

File: docs/minecraft-faq.md [hunk 1]
```diff
@@ -37,3 +46,21 @@
...
+
+
+<a name="add-operators"></a>
+## How do I op myself or my friends?
+
+To op anyone on your serer simply run the /op command in console like follows *(without the `/`)*
+
+`op <playername>`
+
+<a name=""server-cant-keep-up></a>
```
Comment (vaio2005) [inline docs/minecraft-faq.md:62]: Misspell

File: docs/minecraft-faq.md [hunk 2]
```diff
@@ -4,12 +4,21 @@
...
+*   [How do I make it so friends can join?](#port-fowarding)
+*   [How do I op myself or my friends?](#add-operators)
+*   [Server says it cant keep up?](#server-cant-keep-up)
 
 
 <a name="setup-forge"></a>
 ## How do I setup a Forge server?
 
-> WIP
+Forge installation is a bit more complicated than any other server type because forge needs some libraries to run.
```
Comment (vaio2005) [inline docs/minecraft-faq.md:15]: Moved to https://mcserversoft.github.io/documentation/create-forge-server/

**Review by vaio2005 — CHANGES_REQUESTED** [review]
Moved to another location and typo

**Comment by Leonito2003** [comment]
minecraft-faq doesn't exist anymore, and the forge guide has its own page now

**Pull request closed without merging**